common/persistence/nosql/nosqlplugin/interfaces.go

```diff
@@ -76,6 +76,7 @@ type (
 		IsTimeoutError(error) bool
 		IsNotFoundError(error) bool
 		IsThrottlingError(error) bool
+		IsCassandraLWTError(error) bool
```

Comment:
Cassandra-specific func in the nosql interface doesn't look good. Can you give this a more generic name so that it looks less ugly for non-Cassandra nosql plugins? Perhaps something like `IsDBUnavailableError`.